src/commands/src/shared/next-gen-parse.ts

```diff
@@ -18,5 +23,9 @@ export async function nextGenParse({ repoDir, patchId, commitHash }: NextGenPars
     commandArgs.push(patchId);
   }
 
+  if (isProd) {
```

Comment:
IIUC, this is for when we do use this export fn `nextGenParse` for prod, since this is not being used yet. can we add a comment for this?